### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by mbdesigns1989**
Portfolio: Mobile Version
Added "Works" section and "About myself" section.
Used CSS Grid and Flexbox.

### CODE REVIEW

**Comment by mbdesigns1989** [comment]
Please, I'm repeating the week so every review 
count, I don't want to repeat the module, thanks.

File: index.html [hunk 1]
```diff
@@ -4,34 +4,211 @@
...
+    <h2>
+      About <br />
+      Myself
+    </h2>
+    <p>
+      Hello I’m a software developer! I can help you build a product , feature
+      or website Look through some of my work and experience! If you like what
+      you see and have a project you need coded, don’t hestiate to contact me.
+    </p>
+    <a class="contact" href="#" target="_blank">LET'S CONNECT</a>
```
Comment (arslanbisharat) [inline index.html]: - [ ] I suppose you shoudl use hx for this as it's not a button.

File: README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
 # My Portfolio
 
 > This is a Portfolio setup and mobile version  project by microverse to test the use of Flexbox for Positioning and also images and Background,
-![portfolio](./img/my-portfolio.jpg)
+![portfolio](./images/my-portfolio.jpg)
```
Comment (arslanbisharat) [inline README.md:7]: - [ ] [Optional] Live link in your Readme is outdated and it doesn't reflect your new changes. You can update it. 😄

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi @mbdesigns1989, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by arslanbisharat** [comment]
> Please, I'm repeating the week so every review
> count, I don't want to repeat the module, thanks.

Thank you for this message @mbdesigns1989 but you have a few changes I requested earlier. When you are done you are good to go. 
Good job with the project. 👍

**Comment by mbdesigns1989** [comment]
Please, I'm repeating the week so every review 
count, I don't want to repeat the module, thanks.

**Review by bellom — APPROVED** [review]
## Hi @mbdesigns1989  👏  

**Great work making all the necessary corrections from the previous review.**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**